integration-tests/liquibase/pom.xml

```diff
@@ -12,6 +12,12 @@
     <artifactId>quarkus-integration-test-liquibase</artifactId>
     <name>Quarkus - Integration Tests - Liquibase</name>
     <description>Module that contains Liquibase related tests</description>
+
+    <properties>
+        <skipTests>false</skipTests>
+        <skipPMTests>${skipTests}</skipPMTests>
```

Comment:
Those extra executions are a bit annoying when selecting a non-PMT via `-Dtest=...`, but being able to skip them with a property helps.
This is a pattern I could also add to other modules (in a separate PR).